src/themes/dojo/slider.m.css

```diff
@@ -39,20 +39,21 @@
 }
 
 .input {
-	height: var(--border-width);
+	height: 15px;
```

Comment:
Is this component-specific?

Reply:
yeah, I just picked a number that seemed big enough. I'm not sure what variable it would be / if it should even be a variable